conda_forge_tick/make_graph.py

```diff
@@ -203,7 +203,13 @@ def update_nodes_with_bot_rerun(gx):
     for name, node in gx.nodes.items():
         with node["payload"] as payload:
             for migration in payload.get("PRed", []):
-                pr_json = migration.get("PR", {})
+                try:
+                    pr_json = migration.get("PR", {})
+                    # maybe add a pass check info here ? (if using DEBUG)
+                except Exception as e:
+                    logger.error(
+                        f"BOT-RERUN : could not proceed check with {node}, {e}",
+                    )
```

Comment:
I put in `pr_json = None` here just so we always have something asigned to that var.

Reply:
yes, thanks for that. If not, probably the bot would crash again